diff --git a/configure.ac b/configure.ac
index ff70635..eb88f02 100644
--- a/configure.ac
+++ b/configure.ac
@@ -18,10 +18,6 @@
 #                                               -*- Autoconf -*-
 # Process this file with autoconf to produce a configure script.
 
-m4_include([sdk/m4/ax_check_compile_flag.m4])
-m4_include([sdk/m4/ax_check_link_flag.m4])
-m4_include([sdk/m4/ax_cxx_compile_stdcxx.m4])
-m4_include([sdk/m4/ax_atomic.m4])
 
 m4_define([megacmd_major_version], [$(cat src/megacmdversion.h | grep "define MEGACMD_MAJOR" | cut -d" " -f 3)])
 m4_define([megacmd_minor_version], [$(cat src/megacmdversion.h | grep "define MEGACMD_MINOR" | cut -d" " -f 3)])
@@ -42,7 +38,6 @@ AC_PROG_CC
 AC_CONFIG_FILES(Makefile)
 AC_OUTPUT
 
-AC_CONFIG_SUBDIRS(sdk)
 
 
 # Define _GNU_SOURCE
@@ -104,7 +99,6 @@ AM_CONDITIONAL(CPPCHECK, test -n "$HAVE_CPPCHECK")
 AC_LANG_CPLUSPLUS
 
 # Check for C++11 standard
-AX_CXX_COMPILE_STDCXX([11], [noext], [mandatory], [default])
 
 # Check headers
 AC_STDC_HEADERS
@@ -606,7 +600,8 @@ CHECK_ATOMIC()
 #This will simulate compilation of sdk and extract the includes defined for it. Thus we can
 # take care of the configuration done for the subproject withtout having to recheck here
 AC_OUTPUT
-LMEGAINC=$(cd sdk; for i in $(make -n 2>/dev/null | grep "\-I" | grep megaapi ); do echo $i; done | grep "\-I" | sort | uniq | awk '{printf $0" "}')
+PKG_CHECK_MODULES([MEGA],[libmega])
+LMEGAINC=${MEGA_CFLAGS}
 AC_SUBST(LMEGAINC)
 
 # Output
diff --git a/src/include.am b/src/include.am
index 4831549..a0b620c 100644
--- a/src/include.am
+++ b/src/include.am
@@ -13,7 +13,7 @@ endif
 bin_PROGRAMS += $(MEGACMD) $(TESTS)
 
 # establish dependency with libmega
-$(MEGACMD): $(top_builddir)/sdk/src/libmega.la
+$(MEGACMD): $(MEGA_LIBS)
 
 #scripst (to install and where)
 megacmdcompletiondir = $(sysconfdir)/bash_completion.d/
@@ -46,11 +46,11 @@ if USE_PCRE
 src_libmegacmdserver_la_CXXFLAGS += -DUSE_PCRE=1
 endif
 
-src_libmegacmdserver_la_LIBADD = $(DB_LDFLAGS) $(DB_LIBS) $(PCRE_LIBS) $(top_builddir)/sdk/src/libmega.la
+src_libmegacmdserver_la_LIBADD = $(DB_LDFLAGS) $(DB_LIBS) $(PCRE_LIBS) $(MEGA_LIBS)
 
 if !WIN32
 src_libmegacmdserver_la_SOURCES += src/comunicationsmanagerfilesockets.cpp
-noinst_HEADERS += src/comunicationsmanagerfilesockets.h sdk/include/mega/thread/posixthread.h sdk/include/mega/logging.h
+noinst_HEADERS += src/comunicationsmanagerfilesockets.h  
 else
 src_libmegacmdserver_la_SOURCES += src/comunicationsmanagernamedpipes.cpp
 src_libmegacmdserver_la_CXXFLAGS += -D_WIN32=1 -Isdk/include/ -Isdk/include/mega/win32
@@ -84,9 +84,7 @@ noinst_HEADERS += src/megacmdcommonutils.h src/megacmdshell/megacmdshellcommunic
 src_libmegacmdcomms_la_SOURCES = src/megacmdcommonutils.cpp src/megacmdshell/megacmdshellcommunications.cpp
 
 if WIN32
-src_libmegacmdcomms_la_SOURCES += src/megacmdshell/megacmdshellcommunicationsnamedpipes.cpp sdk/src/thread/win32thread.cpp sdk/src/logging.cpp
 else
-src_libmegacmdcomms_la_SOURCES += sdk/src/thread/posixthread.cpp sdk/src/logging.cpp
 endif
 
 src_libmegacmdcomms_la_CXXFLAGS = $(LMEGAINC) $(DB_CXXFLAGS)
@@ -116,6 +114,7 @@ noinst_HEADERS +=
 mega_exec_SOURCES = src/client/megacmd_client_main.cpp
 
 mega_exec_CXXFLAGS = -Isdk/include/ $(LMEGAINC)
+mega_exec_LDADD=$(MEGA_LIBS)
 mega_exec_LDADD = $(top_builddir)/src/libmegacmdclient.la
 ##############################################################################################
 
@@ -124,7 +123,7 @@ noinst_HEADERS += src/megacmdshell/megacmdshell.h
 mega_cmd_SOURCES = src/megacmdshell/megacmdshell.cpp
 
 mega_cmd_CXXFLAGS = $(RL_CXXFLAGS) -Isdk/include/ $(LMEGAINC)
-mega_cmd_LDADD = $(RL_LDFLAGS) $(RL_LIBS) $(TERMCAP_LDFLAGS) $(TERMCAP_LIBS)
+mega_cmd_LDADD = $(RL_LDFLAGS) $(RL_LIBS) $(TERMCAP_LDFLAGS) $(TERMCAP_LIBS)$(MEGA_LIBS)
 mega_cmd_LDADD += $(top_builddir)/src/libmegacmdcomms.la
 
 ##############################################################################################
